tasks/main.yml

```diff
@@ -89,11 +89,47 @@
   become: True
   when: self_signed_certificate|length > 1
 
-
-- name: disable selinux to permit using hosts docker for builds
-  selinux:
-    state: disabled
+- name: set concurrency
+  lineinfile:
+    path:  /srv/gitlab-runner/config/config.toml
+    line: 'concurrent = {{ runner_concurrency }}'
+    regexp: '^concurrent ='
+    state: present
+    insertbefore: 'BOF'
+    create: yes
   become: True
+  when: runner_concurrency is defined
 
 - include: gitlab_runner.yml
```

Comment:
Rename to gitlab_runner_docker.yml for consistency?

Reply:
Yeah, I had that, but tried to minimize the change. I'll change that.